core/src/test/java/org/apache/calcite/rex/RexBuilderTest.java

```diff
@@ -489,6 +490,26 @@ private void checkDate(RexNode node) {
     }
   }
 
+  /** Tests {@link RexBuilder#makeExactLiteral(java.math.BigDecimal)}. */
+  @Test public void testBigDecimalLiteral() {
+    final RelDataTypeFactory typeFactory = new SqlTypeFactoryImpl(RelDataTypeSystem.DEFAULT);
+    final RexBuilder builder = new RexBuilder(typeFactory);
+    assertBigDecimalLiteral(builder, "25");
+    assertBigDecimalLiteral(builder, "9.9");
+    assertBigDecimalLiteral(builder, "0");
+    assertBigDecimalLiteral(builder, "-75.5");
+    assertBigDecimalLiteral(builder, "10000000");
+    assertBigDecimalLiteral(builder, "100000.111111111111111111");
+    assertBigDecimalLiteral(builder, "-100000.111111111111111111");
+    assertBigDecimalLiteral(builder, "73786976294838206464"); // 2^66
+    assertBigDecimalLiteral(builder, "-73786976294838206464");
+  }
+
+  private void assertBigDecimalLiteral(RexBuilder builder, String val) {
+    final RexNode literal = builder.makeExactLiteral(new BigDecimal(val));
+    assertThat(((RexLiteral) literal).getValueAs(BigDecimal.class).toString(), is(val));
```

Comment:
Please add `message` parameter to `assertThat` so it is clear WHAT is test testing

Reply:
Message parameter included.